app/Services/TaskActivityService.php

```diff
@@ -37,13 +32,15 @@ public function getCollection(string $type,int $taskId): array
             $result = $comments;
         }
 
-        foreach ($result as &$item) {
+        $result = array_map(function($item){
             $item['can_change'] = $item['user_id'] === Auth::id();
-        }
+            return $item;
+        },$result);
+
         return $result;
     }
 
-    public function sortCollection(array &$collection, string $sort): void
+    private function _sortCollection(array &$collection, string $sort): void
```

Comment:
PHP EA: You shouldn't use underscore as an indicator of *private* function

Reply:
remove it class